packages/backend/src/paymentPointerKey/routes.ts

```diff
@@ -0,0 +1,36 @@
+import { PaymentPointerContext } from '../app'
+import { PaymentPointerService } from '../open_payments/payment_pointer/service'
+import { PaymentPointerKeyService } from './service'
+
+interface ServiceDependencies {
+  paymentPointerKeyService: PaymentPointerKeyService
+  paymentPointerService: PaymentPointerService
+}
+
+export interface PaymentPointerKeyRoutes {
+  getKeysByPaymentPointerId(ctx: PaymentPointerContext): Promise<void>
+}
+
+export function createPaymentPointerKeyRoutes(
+  deps: ServiceDependencies
+): PaymentPointerKeyRoutes {
+  return {
+    getKeysByPaymentPointerId: (ctx: PaymentPointerContext) =>
+      getKeysByPaymentPointerId(deps, ctx)
+  }
+}
+
+export async function getKeysByPaymentPointerId(
+  deps: ServiceDependencies,
+  ctx: PaymentPointerContext
+): Promise<void> {
+  if (!ctx.paymentPointer) {
+    return ctx.throw(404)
+  }
+
+  const keys = await deps.paymentPointerKeyService.getKeysByPaymentPointerId(
+    ctx.paymentPointer.id
+  )
+
+  ctx.body = keys.map((key) => key.jwk)
```

Comment:
https://auth0.com/docs/secure/tokens/json-web-tokens/json-web-key-set-properties
```suggestion
  ctx.body = {
    keys: keys.map((key) => key.jwk)
  }
```
If we want to include [client interact display info](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-2.3.2) in the `jwks.json` response (instead of the payment pointer query response), it might make sense to move this to `PaymentPointerRoutes` (and remove `PaymentPointerKeyRoutes`)